File: common/src/main/java/com/pluxity/facility/history/FacilityHistoryRepository.java
package com.pluxity.facility.history;

import java.util.List;
import org.springframework.data.jpa.repository.JpaRepository;

public interface FacilityHistoryRepository extends JpaRepository<FacilityHistory, Long> {

    List<FacilityHistory> findByFacilityIdOrderByCreatedAtDesc(Long facilityId);
}

File: common/src/main/java/com/pluxity/facility/history/FacilityHistoryRepository.kt
package com.pluxity.facility.history

import org.springframework.data.jpa.repository.JpaRepository

interface FacilityHistoryRepository : JpaRepository<FacilityHistory?, Long?> {
    fun findByFacilityIdOrderByCreatedAtDesc(facilityId: Long?): MutableList<FacilityHistory?>?
}
